packages/cocoindex/tests/test_provision.py

```
@@ -826,6 +826,160 @@ def test_proxy_failed_state_with_free_port_retries_via_warmup(
+            **SAMPLE_CONFIG,
+            "targets": {
+                "claude": {"enabled": False, "type": "stdio"},
+                "codex": {"enabled": True},
+                "antigravity": {"enabled": True},
+            },
+        }
+        monkeypatch.setattr(provision, "load_package_config", lambda *_: config)
+
+        self._invoke({"cwd": str(tmp_path), "session_id": "sess-target-disable"}, monkeypatch)
+
+        # claude だけエントリが消え、無関係な既存エントリは残る（他は不変）
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)